M2/Macaulay2/packages/Msolve.m2

```diff
@@ -0,0 +1,527 @@
+msolvePresent := run ("type msolve >/dev/null 2>&1") === 0
+-- msolveVersion := if msolvePresent then ... -- no way to get version info!!!
+msolveVersionNeeded := "0.6.4"
+msolvePresentAndModern := msolvePresent -- and match("^[0-9.]+$",msolveVersion) and msolveVersion >= msolveVersionNeeded
+
+newPackage(
+	"Msolve",
```

Comment:
Should this be capitalized as `MSolve` instead? e.g. see the [Sage interface](https://msolve.lip6.fr/downloads/interfaces/msolve-to-sage-file-interface.sage).

Reply:
the msolve team always seems to call it msolve or MSOLVE.... so... I think there is no canonical choice.

Reply:
Other alternative is `msolveInterface` (following `gfanInterface`). Eventually, it might be a good idea to have a subdirectory `packages/interfaces` which contains all interface packages named exactly like the software, e.g. `gfan.m2` `msolve.m2` etc.

Thoughts @mikestillman?